tcms/core/ajax.py

```diff
@@ -100,7 +100,7 @@ def builds(self):
             'is_active': _is_active
         }
 
-        return Build.list(query)
+        return Build.objects.filter(**query)
```

Comment:
1) This makes `Build.list` unused so delete it

2) Instead of using `**query` you can just pass the 2 arguments like regular named parameters to the `.filter()` method.